auacm/app/static/js/controllers/EditProblemController.js

```diff
@@ -65,6 +75,8 @@ app.controller('EditProblemController', ['$scope', '$route', '$http',
             fd.append('sol_file', $scope.solFile);
         if (typeof prob.appeared !== 'undefined')
             fd.append('appeared_in', prob.appeared);
+        if (typeof prob.comp_release !== 'undefined');
+            fd.append('comp_release', parseInt(prob.comp_release));
```

Comment:
You're right. The API serve them back in a GET, I got confused and though we were setting it as well.